cron_inactivity_period/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Inactivity Periods for Cron Jobs',
+    'version': '8.0.1.0.0',
+    'author': "GRAP,Odoo Community Association (OCA)",
```

Comment:
space

Reply:
no. Not a good idea to add space ! it had broken Travis.
https://travis-ci.org/OCA/server-tools/jobs/420041893#L1105
regards.